From 73e94f5b926d1395cf058ad5e16e5a701f538b7e Mon Sep 17 00:00:00 2001
From: tanyulong <tanyulong@kylinos.cn>
Date: Tue, 7 Dec 2021 15:06:16 +0800
Subject: [PATCH] Modified fifo path

---
 libsearch/file-utils.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libsearch/file-utils.h b/libsearch/file-utils.h
index c47f225..5ae32e7 100644
--- a/libsearch/file-utils.h
+++ b/libsearch/file-utils.h
@@ -55,6 +55,7 @@
 //#define CREATING_INDEX 1
 //#define FINISH_CREATING_INDEX 2
 #define MAX_CONTENT_LENGTH 20480000
+#define UKUI_SEARCH_PIPE_PATH (QDir::homePath()+"/.config/org.ukui/ukui-search/ukuisearch").toLocal8Bit().constData()
 
 namespace Zeeker {
 class  LIBSEARCH_EXPORT FileUtils {
-- 
2.30.0

